src/Components/InputSection/components/Output.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useContext } from "react";
+import { GUESSES } from "../../../constants";
+import GameContext from "../../../Contexts/gameContext";
+import { BsCircleFill, BsXCircle, BsCircle } from "react-icons/bs";
+
+type PropType = {
+  index: number;
+};
+
+const Output: React.FunctionComponent<PropType> = ({ index }) => {
+  const { state } = useContext(GameContext);
+  const output = state.output[index];
+
+  return (
+    <div className="output">
+      <div className="output-row1">
+        {GUESSES.map((guessNo) => {
+          return (
+            <div key={guessNo} className="output-circle">
+              {output[guessNo] === 1 ? (
```

Comment:
lets use enum instead of 1, -1
since it is difficult to know what 1, -1 is for

Reply:
done